subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -337,7 +337,11 @@ static MFIFO_DEFINE(prep, sizeof(struct lll_event), EVENT_PIPELINE_MAX);
  */
 #if !defined(VENDOR_EVENT_DONE_MAX)
 #if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_OBSERVER)
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
```

Comment:
Can `CONFIG_BT_CTLR_PHY_CODED` be enabled when `CONFIG_BT_CTLR_ADV_EXT` and `CONFIG_BT_OBSERVER` is disabled? Just making sure whether we need to increment the `#define EVENT_DONE_MAX 4` in that case.

Reply:
It can be enabled for Advertising Extensions support in Broadcaster state, but the increase in `EVENT_DONE_MAX` is only required in Observer state where Scanning can be enabled simultaneous on 1M and Coded PHY.

UIpdate the comments above accordingly.